learning/getting-started.md

```diff
@@ -0,0 +1,35 @@
+
+## Getting Packages
+When you downloaded Julia using the instructions above, you got the compiler (which converts the Julia code you write into a version your computer can run), and the standard library (also sometimes called *Base*), which contains basic functionality that comes built in. This includes things like arrays and lists, numbers and strings,  some basic linear algebra and statistics etc. For most use cases, you will likely need to download and run external libraries, which are known as packages in Julia. A list of publicly available packages is [available here](https://pkg.julialang.org)
+
+Packages are downloaded using a package manager built into the Julia standard library. You can interact with the package manager in a couple of ways. The simplest is to press `]` (the right square bracket) on your keyboard when you are at the REPL, which will put the REPL into _pkg_ mode. In this mode, you type package management commands directly into the REPL. For example, `add PackageName` will add a package to your local environment, and `rm PackageName` will remove it. Using the package manager will also typically manage the package’s dependencies for you; this means the package manager makes sure your computer/environment has everything it needs in order to run the package you want to install. A [video](https://www.youtube.com/watch?v=76KL8aSz0Sg) showing how to use the package manager in more detail can be seen on our youtube channel.
+
+## Coming from other languages
+If you have experience programming in another language, you will find that most of your knowledge will be easily transferred to Julia. We have documented some [noteworthy differences](https://docs.julialang.org/en/v1/manual/noteworthy-differences/) from other popular languages: [Matlab](https://docs.julialang.org/en/v1/manual/noteworthy-differences/#Noteworthy-differences-from-MATLAB-1), [R](https://docs.julialang.org/en/v1/manual/noteworthy-differences/#Noteworthy-differences-from-R-1), [Python](https://docs.julialang.org/en/v1/manual/noteworthy-differences/#Noteworthy-differences-from-Python-1) and [C/C++](https://docs.julialang.org/en/v1/manual/noteworthy-differences/#Noteworthy-differences-from-C/C-1)
+
+
+## Developing packages
+It is easy to move from using Julia packages to developing them. Most of our contributors started that way. If you are the user of a Julia package, please believe that you have the skills to contribute bug fixes and new features. Chris Rackaukas has created a video that walks you through the process of creating a new package, or contributing code to an existing package. Find it [here on our Youtube channel](https://www.youtube.com/watch?v=QVmU29rCjaA)
+

```

Comment:
```suggestion


## Looking for help

Programming is fun and exciting. The Julia programming language is no exception. While it is always best to try to answer a question yourself if you can, feel free to ask someone else more experienced! Take a look at the [Julia Community](https://julialang.org/community/) tab on the website to find all the places you can connect and get help in the Julia Community. 
```

Reply:
May also be worth adding a link to the FAQ section of the docs: https://docs.julialang.org/en/v1/manual/faq/

Reply:
Borrowing these ideas from: https://www.python.org/about/gettingstarted/